settings.js

```diff
@@ -5,7 +5,7 @@
 var apiHost = process.env.API_HOST || 'http://localhost:8084';
 var atlasWebComponentsUrl = process.env.ATLAS_WEB_COMPONENTS_URL;
 var authEndpoint = process.env.AUTH_ENDPOINT || apiHost + '/auth/user';
-var authEnabled = process.env.AUTH_ENABLED === 'false' ? false : true;
+var authEnabled = process.env.AUTH_ENABLED === 'true' ? true : false;
```

Comment:
```suggestion
var authEnabled = process.env.AUTH_ENABLED === 'true';
```
Would this be equivalent?

Reply:
That's quite true. I'll change it as you expect. But how about the others, like l20 (`const entityTagsEnabled = process.env.ENTITY_TAGS_ENABLED === 'true' ? true : false;`)?

Reply:
No need to update the rest of the file; just thought it might be more readable without the ternary since we're making an edit to this line anyway!

Reply:
I agree with you. Have updated that. Thank you 😄